code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -761,6 +761,7 @@ var/list/ai_list = list()
 		radio_enabled = 0 	//No talking on the built-in radio for you either!
 		loc = card//Throw AI into the card.
 		card.AI = src
+		user.forceMove(src)

```

Comment:
replace the line two above

    loc = card

with 

   forceMove(card) //basically the same as src.forceMove(card), also don't include this comment